app/javascript/app/components/my-climate-watch/my-account/my-account-component.jsx

```diff
@@ -1,16 +1,148 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import LoginProvider from 'providers/login-provider';
+import TextInput from 'components/text-input';
+import Dropdown from 'components/dropdown';
+import Button from 'components/button';
 
-const MyAccount = ({ user }) => (
-  <div>
-    <h1>User details</h1>
-    <span>Email:</span>
-    <span>{user.email}</span>
-  </div>
-);
+import theme from 'styles/themes/input/text-input-theme.scss';
+import styles from './my-account-styles.scss';
+
+const MyAccount = props => {
+  const {
+    email,
+    firstName,
+    lastName,
+    organization,
+    sector,
+    country,
+    countries,
+    sectors,
+    dataUsage,
+    tester,
+    isProfileUpdated,
+    updateUserFirstName,
+    updateUserLastName,
+    updateUserOrganization,
+    updateUserSector,
+    updateUserCountry,
+    updateUserDataUsage,
+    updateUserTester,
+    saveUserData
+  } = props;
+
+  return (
+    <div>
+      <LoginProvider />
+      <h1 className={styles.title}>User Profile</h1>
+      <div className={styles.personalInfo}>
+        <TextInput
+          className={styles.input}
+          theme={theme}
+          placeholder={'Add a name'}
+          label={'First name'}
+          inputType={'text'}
+          onBlur={updateUserFirstName}
+          value={firstName}
```

Comment:
We could add a value={firstName || ''} to avoid the warning message. The input doesn't like null or undefined as a value